### ISSUE TITLE
Support for Python 3.11-dev

### ISSUE THREAD

**Issue by FFY00** [actions/setup-python#251]
We have an alpha release, it would be very helpful to be able to start testing against it :blush:

https://github.com/python/cpython/releases/tag/v3.11.0a1
https://github.com/python/cpython/tags

**Comment by henryiii** [comment]
This is out according to actions/python-versions!

**Comment by MaksimZhukov** [comment]
Hello @FFY00! Thank you for the request!
We've published Python 3.11.0-alpha.1, so you can test it using the following snippet: 
```
    steps:
    - name: Setup Python 3.11.0-alpha.1
      uses: actions/setup-python@v2
      with:
        python-version: '3.11.0-alpha.1'
```

I'm closing the issue. Feel free to contact us if you have any questions.
Thanks!

**Comment by SamuelMarks** [comment]
The range syntax also works, will bring in the latest: '3.10.0-alpha - 3.10.0' 

Like so:
```yaml
python-version: [3.6, 3.7, 3.8, 3.9, '3.10', '3.11.0-alpha - 3.11.0']
```
